apps/dashboard/src/components/auth/auth-feature-row.tsx

```diff
@@ -0,0 +1,19 @@
+import { ReactNode } from 'react';
+
+interface AuthFeatureRowProps {
+  icon: ReactNode;
+  title: string;
+  description: string;
+}
+
+export function AuthFeatureRow({ icon, title, description }: AuthFeatureRowProps) {
+  return (
+    <div className="inline-flex items-start justify-start gap-3.5">
+      <div className="flex items-center justify-center p-1">{icon}</div>
+      <div className="inline-flex shrink grow basis-0 flex-col items-start justify-start gap-2">
+        <div className="text-sm font-medium text-[#0d111b]">{title}</div>
+        <div className="text-xs font-medium text-[#99a0ad]">{description}</div>
```

Comment:
@LetItRock what is the suggested way of working with brand colors and typography? do we have some class names for those?

Reply:
Brand colors should be defined in HSL format in the [index.css](https://github.com/novuhq/novu/blob/next/apps/dashboard/src/index.css#L52) file, they should then be made available to Tailwind by adding an entry referencing the variable definition in the [tailwind.config.js](https://github.com/novuhq/novu/blob/next/apps/dashboard/tailwind.config.js#L65) file